Gemfile

```diff
@@ -41,6 +41,13 @@ group :development, :test do
   gem 'jshint'
 end
 
+group :test do
+  gem 'rspec'
+  gem 'rspec-rails'
+  gem 'guard-rspec'
+  gem 'capybara', '2.5.0'
```

Comment:
Seems like a good idea to have non-testing/dev-only Gems with explicit versions. Any reason to explicitly add a version to Capybara and not doing so for the rspec gems?